snapcraft.tpl.yaml

```
@@ -24,6 +24,16 @@ parts:
     build-packages:
        - pkg-config
 
+plugs:
+  dot-local-share-typst:
+    interface: personal-files
+    write:
```

This directory only needs read access
```suggestion
    read:
```